Mathlib/Algebra/FreeMonoid/Symbols.lean

```diff
@@ -0,0 +1,35 @@
+/-
+Copyright (c) 2024 Hannah Fechtner. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Hannah Fechtner
```

Comment:
```suggestion
Authors: Hannah Fechtner
```
I think that this is one of the linter's complaints.

Reply:
yup :) before it wasn't even getting to that, so this is some progress, at least!